CarInspection/src/WebCarInspection/Helpers/ApiClientHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.Extensions.Options;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using WebCarInspection.Interfaces;
+using WebCarInspection.ServerInfo;
+
+namespace WebCarInspection.Helpers
+{
+    public class ApiClientHelper : IApiClientHelper
+    {
+        private readonly HttpClient _client;
+        private readonly IOptions<ApiServerInfo> _apiServerInfo;
+
+        public ApiClientHelper(HttpClient client,
```

Comment:
Register HttpClient in startup with baseUrl and use it here

Reply:
https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests